app/presenters/curation_concerns/presents_attributes.rb

```diff
@@ -39,11 +40,24 @@ def microdata_type_to_html
 
     private
 
-      def renderer_for(field, options)
-        if options[:catalog_search_link]
-          Renderers::LinkedAttributeRenderer
-        elsif field == :rights
-          Renderers::RightsAttributeRenderer
+      def find_renderer_class(name)
```

Comment:
I fell into a trap earlier in that `#const_get` seems to trigger autoloading, while `#const_defined?` doesn't. I couldn't figure out a reasonable way around it that didn't depend on `begin ... rescue NameError`